src/Paket.Core/Utils.fs

```diff
@@ -295,11 +344,15 @@ let envProxies () =
         if isNull envVarValue then None else
         match Uri.TryCreate(envVarValue, UriKind.Absolute) with
         | true, envUri ->
+#if NETSTANDARD1_6
+            raise <| System.NotImplementedException ("I don't know how WebProxy can should be replace in dotnetcore. Therefore this is currently not supported. Please implement me :)")
```

Comment:
They changed this API so much I have no clue how this is done in netstandard, sorry ...
For me (not using proxies) this is fine for now. And I couldn't test either.